app/views/devise/mailer/invitation_instructions.html.erb

```diff
@@ -39,6 +39,11 @@
         If you are ready to get started, please set your password. This is the first step to accessing your new account.
       </td>
     </tr>
+    <tr style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; margin: 0;">
+      <td class="content-block" style="font-family: Helvetica,Arial,sans-serif; box-sizing: border-box; font-size: 14px; vertical-align: top; margin: 0; padding: 0 0 20px;" valign="top">
```

Comment:
We shouldn't use CSS class in mailers

Reply:
it's ok. I don't think the class does anything